src/sessionTypes/index.spec.js

```diff
@@ -0,0 +1,102 @@
+import auth0 from 'auth0-js';
+import Auth0WebAuth from './auth0WebAuth';
+import PasswordGrantAuth from './passwordGrantAuth';
+import MachineAuth from './machineAuth';
+import * as sessionTypes from './index';
+
+describe('sessionTypes', function() {
+  afterEach(function() {
+    sinon.restore();
+  });
```

Comment:
Is there no need for a `beforeEach` here to set up `sinon`?

Reply:
Zach pointed out a change in `sinon` about a month ago, where newer versions have a default sandbox set up already! https://github.com/ndustrialio/contxt-sdk-js/pull/120#issuecomment-502125529

Reply:
Cool cool cool

Reply:
This is cool!